main.py

```
@@ -0,0 +1,166 @@
+Lunar Lander TUI Game
+
+A simple text-based lunar lander game using curses for the terminal interface.
+"""
+
+import curses
+import sys
+import time
+from game import constants
+
+
+def check_terminal_size(stdscr):
```

Ruff (0.8.2)

Missing return type annotation for public function `check_terminal_size`

(ANN201)